### DISCUSSION TITLE
Solution to add a [copy to clipboard] button in code fence style?

### DISCUSSION THREAD

**Discussion by tsunashitmt** [Milkdown/milkdown#414] [Q&A]
Hi Guys,
Seems a lot wiki platform support such feature to display a [copy to clipboard] button in code blocks.
Is there a best practice for milkdown's code fence achieve such function?
I tried to insert a DOM with button tag integrated clipboard.js after content updated but with no luck, and extend code-fence.ts seems quite hard for me, any sample code would be appreciate.

**Comment by Saul-Mirone** [comment]
The milkdown@v6 will make you can extend the view of code fence(any node in fact). I'm working on it.

**Reply by tsunashitmt** [reply]
Thanks you for your reply.
I am looking forward the @v6 release
